src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        Color randomColor = colorSupplier.getRandomColor();
+        int minNumber = 1;
+        int maxNumber = 100;
```

Comment:
These numbers should be constants
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)